tests/unit-tests/UnitTests/Controllers/PreviewControllerTests.cs

```diff
@@ -66,6 +66,7 @@ public async Task IndexPost_ShouldReturn_View_IfUseGeneratedViewModel_IsTrue()
             var result = await _controller.IndexPost(new List<CustomFormFile>());
 
             _previewService.Verify(_ => _.VerifyPreviewRequest(It.IsAny<List<CustomFormFile>>()), Times.Once);
+
```

Comment:
As both 68 and 70 are part of the // Assert I don't think we need the empty line between them